pages/give-devices.js

```diff
@@ -1,50 +1,104 @@
 import Head from 'next/head';
 import React from 'react';
+import { useRouter } from 'next/router';
 
 import Footer from '../components/Footer';
 import InputFormControl from '../components/form/InputFormControl';
+import Modal from '../components/Modal';
 import Navbar from '../components/Navbar';
 import SelectFormControl from '../components/form/SelectFormControl';
 import TextareaFormControl from '../components/form/TextareaFormControl';
 
+// Use ISO 3166 country codes per https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
 const COUNTRIES = [
-  { value: 'united-states', label: 'United States' },
-  { value: 'canada', label: 'Canada' },
-  { value: 'mexico', label: 'Mexico' },
-  { value: 'germany', label: 'Germany' },
-  { value: 'latvia', label: 'Latvia' },
-  { value: 'india', label: 'India' },
-  { value: 'bangladesh', label: 'Bangladesh' },
-  { value: 'philippines', label: 'Philippines' },
+  { value: 'US', label: 'United States' },
+  { value: 'CA', label: 'Canada' },
+  { value: 'MX', label: 'Mexico' },
+  { value: 'DE', label: 'Germany' },
+  { value: 'LV', label: 'Latvia' },
+  { value: 'IN', label: 'India' },
+  { value: 'BD', label: 'Bangladesh' },
+  { value: 'PH', label: 'Philippines' },
 ];
 
 export default function GiveDevicesPage() {
+  const router = useRouter();
+  const [modalText, setModalText] = React.useState('');
+  const [isModalOpen, setModalOpen] = React.useState(false);
   const [name, setName] = React.useState('');
   const [email, setEmail] = React.useState('');
-  const [country, setCountry] = React.useState(COUNTRIES[0].id);
+  const [country, setCountry] = React.useState(COUNTRIES[0].value);
   const [streetAddress, setStreetAddress] = React.useState('');
+  const [streetAddress2, setStreetAddress2] = React.useState('');
   const [city, setCity] = React.useState('');
   const [state, setState] = React.useState('');
   const [postalCode, setPostalCode] = React.useState('');
-  const [phoneNumber, setPhoneNumber] = React.useState('');
-  const [valueOfDevice, setValueOfDevice] = React.useState('');
-  const [about, setAbout] = React.useState('');
+  const [phone, setPhone] = React.useState('');
+  const [originalPurchasePrice, setOriginalPurchasePrice] = React.useState('');
```

Comment:
Thanks for changing `valueOfDevice` to `originalPurchasePrice`. This helps us follow best practices for donation valuation for better tax compliance.